### DISCUSSION TITLE
Background Service

### DISCUSSION THREAD

**Discussion by nikhil-alr** [talsec/Free-RASP-Community#27] [Q&A] [answered]
With Free-Rasp I can see some code getting invoked periodically to report callbacks.
Can you tell me how much it's affecting battery drain and performance?

Thanks

**Comment by syakymchuk** [comment, marked as answer]
Hello @nikhil-alr, 

The battery draining is not very intensive. The SDK makes background checks every 3 minutes. There is no intensive computation or intensive internet connectivity.
